### ISSUE TITLE
I'm getting messages from user not in my friendlist

### ISSUE THREAD

**Issue by roughnecks** [bitlbee/bitlbee-steam#130]
Hi.

I've been trading with scrap.tf bots recently and every time I want to buy something, their bot sends me the trade offer along with a text message, which I receive via bitlbee only.
The sender's nickname is composed by an underscore, followed by its steam ID.

How can they do that, if we're not friends on Steam?

**Comment by AiyionPrime** [comment]
https://steamcommunity.com/app/440/discussions/0/2765630416818067729/
They may message you by beeing in the same group.

**Comment by roughnecks** [comment]
I haven't joined any group.

**Comment by AiyionPrime** [comment]
And youre sure, youre not friends with any of their bots?
Like when you get gifts or trade and stuff you become friends automagically?

**Comment by roughnecks** [comment]
Yeah, I'm sure. BTW there's no need to be friends to trade and I don't think scrap.tf gifts anything.
